students.rb

```diff
@@ -0,0 +1,10 @@
+# Named Students to differentiate it from the already existing Student class
+class Students
```

Comment:
- [ ] [OPTIONAL] 
A _student_ object inherits from the _person_ class. Let's endeavor to declare this relationship.